invenio_app_rdm/theme/assets/semantic-ui/js/invenio_app_rdm/landing_page/RecordCommunitiesListModal/RecordCommunitiesSearch.js

```diff
@@ -54,29 +54,32 @@ export class RecordCommunitiesSearch extends Component {
           searchApi={searchApi}
           initialQueryState={{ size: 5, page: 1 }}
         >
-          <Container fluid>
-            <Container fluid>
+          <>
+            <Modal.Content>
```

Comment:
I am a bit confused, why is it changed to a modal? it was already inside one

Reply:
Because the containers are causing a lot of horizontal spacing on mobile.